apify-docs-theme/src/theme/custom.css

```diff
@@ -74,6 +159,20 @@ html[data-theme='dark'] {
 
     --ifm-heading-color: #242736;
 
+    --ifm-leading: calc(var(--ifm-leading-desktop) * 1.6rem);
+    --ifm-list-margin: 1.6rem;
+    --ifm-pre-padding: 1.6rem;
+
+    --ifm-hr-margin-vertical: 2.4rem;
+
+    /* Comes from the Apify design theme content typography tokens */
+    --ifm-h1-font-size: 2.4rem;
+    --ifm-h2-font-size: 2rem;
```

Comment:
These don't get applied in the docs. Even when I tried adding `!imporant` the font-size is still overridden by `.markdow`

![image](https://github.com/apify/apify-docs/assets/77627332/ba636732-0360-4beb-a754-bdcf71548437)